index.html

```diff
@@ -35,6 +35,149 @@ <h1 class="name">I’m Tarik Glad to see you!</h1>
+                </ul>
+                <div><button class="btn">See Project</button></div>
+            </div>
+        </div>
+     </section>
+
+     <!--About me Section-->
+    <section class="about-me">
+        <div class="personal-info">
+            <h2>About Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="blue text">LET’S CONNECT</p>
+            <div>
+                <ul class="social-media">
+                    <li><a href="https://twitter.com/TarikBouari"><img class="icon" src="images/Vector2.png" alt="twitter"></a></li>
+                    <li><a href="https://www.linkedin.com/in/tarik-bouari-44b7191a6/"><img class="icon" src="images/Linkedin.png" alt="twitter"></a></li>
+                    <li> <a href="#"><img class="icon" src="images/VectorM.png" alt="twitter"></a></li>
+                    <li> <a href="https://github.com/tarikbouari"><img class="icon" src="images/Vector3.png" alt="twitter"></a></li>
+                    <li><a href="#"><img class="icon" src="images/Vector5.png" alt="twitter"></a></li>
+                </ul>    
+            </div>
+            <div><button class="btn">Get my resume</button></div>
+        </div>
+        <div class="code-lang">
+            <ul>
+                <li>
+                    <div>
+                        <div class="lang">
+                            <h3>Languages</h3>
+                            <img src="images/angle-down.svg" class="arrow icon" alt="arrow-down">
+                        </div>
+                        <div class="lang-child">
+                            <div class="js">
+                                <img src="images/java.png " class="img-lang" alt="javascript">
+                                <p>JavaScript</p>
+                            </div>
+                            <div class="html">
+                                <img src="images/html.png" class="img-lang" alt="html">
+                                <p>HTML</p>
+                            </div>
+                            <div class="css">
+                                <img src="images/css.png" class="img-lang" alt="">
+                                <p>CSS</p>
+                            </div>
+                        </div>     
+                    </div>
+                </li>
+                <li>
+                    <div class="lang">
+                        <h3>Frameworks</h3>
+                        <img src="images/angle-right.svg" class="arrow icon" alt="arrow-right">
+                    </div>
+                </li>
+                <li>
+                    <div class="lang">
+                        <h3>Skills</h3>
+                        <img src="images/angle-right.svg" class="arrow icon" alt="arrow-down">
```

Comment:
- [ ] It happens that your arrows here are not displaying as it shows on the figma design. I looked into it and found that there's a padding bottom being applied to them with CSS. Kindly fix that. 

##### Figma Design
![image](https://user-images.githubusercontent.com/29071984/176786923-32bdf097-2e47-4a53-a2e5-b9e8aeb007fe.png)

##### Your Design
![image](https://user-images.githubusercontent.com/29071984/176786512-46736a7e-26f9-4be5-afbf-aac30827c022.png)